### PR TITLE
out_s3: added static file path documentation

### PR DESCRIPTION
**Pull request by StephenLeeY**
Signed-off-by: Stephen Lee <sleemamz@amazon.com>

### CODE REVIEW

File: pipeline/outputs/s3.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Records are stored in files in S3 as newline delimited JSON.
...
 | s3\_key\_format\_tag\_delimiters | A series of characters which will be used to split the tag into 'parts' for use with the s3\_key\_format option. See the in depth examples and tutorial in the documentation. | . |
+| static\_file\_path | Disables behavior where UUID string is automatically appended to end of S3 key name when `$UUID` is not provided in `s3_key_format`. If this is enabled without a dynamic key format, the previous file will be overwritten. | false |
```
Comment (oleglypkan) [inline pipeline/outputs/s3.md:30]: @StephenLeeY What is the behavior when ```$UUID``` or ```$TAG``` or date/time specific formatters (```%Y/%m/%d/%H/%M/%S```) are specified and ```static_file_path``` is set to ```true```? Will it have any effect in this case? I would clarify it in the documentation.

**Review by oleglypkan — APPROVED** [review]

**Comment by edsiper** [comment]
pls fix latest conflict so we can merge this PR

**Comment by lecaros** [comment]
Hey @PettitWesley, could you please also take a look into this one?

**Comment by PettitWesley** [comment]
https://github.com/fluent/fluent-bit-docs/pull/673

**Pull request closed without merging**